caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters for the entire run of initialization and OnACID
+
+            estimates: Estimates, optional
+                Estimates object to load an existing model
+
+            path: str, optional
+                path to a saved OnACID model on disk
+
+            dview:
+                dview instance
```

Comment:
Needs a bit more explanation

Reply:
I describe it as a "multicore processing object" and think that might be a good generic description?

It would be nice if we had better general documentation on `dview` (in the `setup_cluster()` the docs for it are "dview: ipyparallel dview object, or for multiprocessing: Pool object". But if the users use SLURM I'm not sure what dview will return. I think "multicore processing object" or something similar will work for all use cases?.

Reply:
yea the name dview isn't great and I myself am not sure exactly what this is in different scenarios, feel free to edit/suggest